lighthouse-core/gather/gatherers/dobetterweb/geolocation-on-start.js

```diff
@@ -23,6 +23,15 @@
 
 const Gatherer = require('../gatherer');
 
+/* global navigator */
+
+/* istanbul ignore next */
+function queryGeolocationPermission() {
+  return navigator.permissions.query({name: 'geolocation'}).then(result => {
```

Comment:
thats very cool. do we need to do a feature detect before using this or are we safe?